### PR TITLE
cpio: add page

### PR DESCRIPTION
**Pull request by FlyingJester**

### CODE REVIEW

**Comment by waldyrious** [comment]
Thanks! Any chance you could integrate what each option short name stands for into the example descriptions?

**Comment by FlyingJester** [comment]
Yes, I can.

**Comment by FlyingJester** [comment]
Are there any good examples already in the pages of this I can use for reference?

**Comment by waldyrious** [comment]
I believe this is present in most pages (or at least I'd hope so -- I didn't conduct an exhaustive survey) but for example you could look at [sed](https://github.com/tldr-pages/tldr/blob/master/pages/common/sed.md): note how `-r` is matched by "replace" in the description, and how `-i` was integrated into the description to make sure it was explained. Other good examples of discussions about this can be seen [here](https://github.com/tldr-pages/tldr/pull/870#discussion-diff-62154910) and [here](https://github.com/tldr-pages/tldr/pull/827#discussion-diff-56882383). Let me know if that helps.

**Comment by FlyingJester** [comment]
I think my examples kind of already do that, though. `i` is for input, `o` is for output. I could remove or separately explain the `v` for verbose, if that would help. `cpio` doesn't have long options, except for a couple which are kind of unusual to actually use (not for the basic stuff like what is shown in these examples).

I can change the descriptions to explicitly say 'input' and 'output', and make a separate `v` example (or mention that it's for verbose mode).

**Comment by waldyrious** [comment]
> I can change the descriptions to explicitly say 'input' and 'output', and make a separate `v` example (or mention that it's for verbose mode).

Yes, that'd be great!

File: pages/common/cpio.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+> Archiving utility.
+
+- Create an archive from files:
+
+`echo "{{file1}} {{file2}} {{file3}}" | cpio -ov > {{archive.cpio}}`
+
+- Create an archive from a directory:
+
+`find {{directory}} | cpio -ov > {{archive.cpio}}`
```
Comment (agnivade) [inline pages/common/cpio.md]: {{directory}} => {{path/to/directory}}

**Comment by agnivade** [comment]
Hey @FlyingJester , any updates on this ?

**Pull request closed without merging**